include/lsst/meas/base/PixelFlags.h

```diff
@@ -42,13 +41,20 @@ namespace lsst { namespace meas { namespace base {
  */
 class PixelFlagsControl {
 public:
-
+    LSST_CONTROL_FIELD(center, std::vector<std::string>,
+                       "List of mask planes for which to search center of footprint");
+    LSST_CONTROL_FIELD(any, std::vector<std::string>,
+                       "List of mask planes for which to search entire footprint");
```

Comment:
I find those docstrings a bit... lacklustre. Unless you already know in detail how the algorithm works, you wouldn't be any the wiser from reading them. Can we be any more descriptive?

(Actually, this whole algorithm needs proper documentation, but I think that's outside the scope of this ticket.)

Also, the names `any` and `center` are a bit too generic (below when we start referring to `control.any` it's pretty opaque).